app/models/user.rb

```diff
@@ -1,3 +1,12 @@
 class User < ApplicationRecord
-  validates_presence_of :uid, :token
```

Comment:
Removed validations from first model spec to make a new user model which will need a new spec